src/resources/postcss/tribe-common-admin.pcss

```diff
@@ -497,6 +497,13 @@ table.plugins .tribe-plugin-update-message a {
 	margin-left: 25%;
 	padding-left: 10px;
 }
+.system-info-copy .system-info-copy-btn {
+	padding: 6px;
+}
+
+.system-info-copy .system-info-copy-btn .dashicons {
```

Comment:
This block should be re-written as follows to leverage the nesting we get with PostCSS:

```
.system-info-copy {
  .system-info-copy-btn {
    padding: 6px;

    .dashicons {
      padding-right: 10px;
    }
  }
}
```

Adding a line break after the `.template-updates-wrapper dd { ... }` block for readability might be a good plan.